plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestBackgroundHiveSplitLoader.java

```diff
@@ -348,7 +348,7 @@ public TupleDomain<ColumnHandle> getCurrentPredicate()
     public void testCachedDirectoryLister()
```

Comment:
We should add tests for ensuring the right files get listed depending on the filter

Reply:
added here - lib/trino-filesystem-s3/src/test/java/io/trino/filesystem/s3/TestS3FileSystemAwsS3.java